plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -1151,6 +1166,80 @@ private void testCreateTableColumnMappingMode(ColumnMappingMode mode, Consumer<S
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test(dataProvider = "columnMappingModeDataProvider")
+    public void testDropAndAddColumnShowStatsForColumnMappingMode(ColumnMappingMode mode)
+    {
+        if (mode == ColumnMappingMode.NONE) {
+            throw new SkipException("The connector doesn't support dropping columns with 'none' column mapping");
```

Comment:
nit: It is actually the Delta table format which doesn't support this operation.